### PR TITLE
Houlahan/bugs/media

### PR DESCRIPTION
**Pull request by randyhoulahan**

### CODE REVIEW

File: app/views/kronos/participation-form.js [hunk 1]
```diff
@@ -233,38 +233,40 @@ define(['app', 'services/conference-service','providers/locale','directives/kron
...
-                  if(newValue._id != oldValue._id){
-                    _ctrl.doc.focalPoint = newValue._id
-                    save()
-                  }
-                },true)
+      if(_ctrl.doc.nominatingOrganization)
+        return getOrg().then(function(){
+          var headPromise  = getHead()
+          var focalP = getFocalPoint()
+          return Promise.all([headPromise,focalP]).then(function(){
```
Comment (blaisefonseca) [inline app/views/kronos/participation-form.js]: $q.all([])

File: app/views/kronos/participation-form.js [hunk 2]
```diff
@@ -233,38 +233,40 @@ define(['app', 'services/conference-service','providers/locale','directives/kron
...
+            $timeout(function(){
```
Comment (blaisefonseca) [inline app/views/kronos/participation-form.js]: no need for timeout

File: app/views/kronos/participation-form.js [hunk 3]
```diff
@@ -233,38 +233,40 @@ define(['app', 'services/conference-service','providers/locale','directives/kron
...
+                  $scope.$watch('participationCtrl.head',function(newValue, oldValue){
+                    if(newValue._id != oldValue._id){
+                      _ctrl.doc.head = newValue._id
+                      save()
+                    }
+                  },true)
+                  $scope.$watch('participationCtrl.focalPoint',function(newValue, oldValue){
+                    if(newValue._id != oldValue._id){
+                      _ctrl.doc.focalPoint = newValue._id
+                      save()
```
Comment (blaisefonseca) [inline app/views/kronos/participation-form.js]: does this mean auto save?

Comment (randyhoulahan) [reply app/views/kronos/participation-form.js]: not exactly in the way you are thinking but yes.  Its needed in this case.

File: app/directives/file.html [hunk 4]
```diff
@@ -5,8 +5,8 @@
         
         <span ng-if="caption">{{caption}}</span>
         <span ng-if="!caption">
-            <span ng-if="!isMutiple()"><strong>Choose a file</strong> <span ng-if="allowDragDrop">or drag it here</span></span>
-            <span ng-if=" isMutiple()"><strong>Choose files</strong>  <span ng-if="allowDragDrop">or drag them here</span></span>
+            <span ng-if="!isMutiple()"><strong>Choose a file</strong><span ng-if="!allowDragDrop">by clicking here</span> <span ng-if="allowDragDrop">or drag it here</span></span>
+            <span ng-if=" isMutiple()"><strong>Choose files</strong>  <span ng-if="!allowDragDrop">by clicking here</span><span ng-if="allowDragDrop">or drag them here</span></span>
```
Comment (blaisefonseca) [inline app/directives/file.html]: u need space  before or

**Pull request merged**